configuration/rules-dsl.md

```diff
@@ -90,19 +90,17 @@ The **Rules** section contains a list of rules.
 Each rule has the following syntax:
 
 ```java
-rule "rule name"
+rule "<RULE_NAME>"
 when
-    <TRIGGER CONDITION1> or
-    <TRIGGER_CONDITION2> or
-    <TRIGGER_CONDITION3>
-    ...
+    <TRIGGER_CONDITION>
 then
     <SCRIPT_BLOCK>
 end
 ```
 
-A rule can have any number of trigger conditions, but must at least have one.
-The `SCRIPT_BLOCK` contains the code that should be executed, when a trigger condition is met, see the [script](#scripts) section for details on its syntax.
+- `<RULE_NAME>` - Each rule must have a unique name (given within quotes). It is recommended to choose a meaningful description as rule name.
+- `<TRIGGER_CONDITION>` - The triggering event upon which the rule logic is executed. At least one condition has to be given, multiple conditions are separated by the keyword `or`. Please see below for different possible triggers.
```

Comment:
Based upon @ThomDietrich's comments, please consider the following.  "<TRIGGER_CONDITION>- A rule is executed in reaction to one or more trigger conditions.  Please see below for possible triggers."

Reply:
I'd leave the first sentence, or at least it's notion.
It is there to clearly state WHAT trigger condition is.
After that sentence you may describe as much as you want or all I care ;)

Reply:
Agree completely.